client/lib/main.dart

```diff
@@ -30,13 +30,13 @@ class MyApp extends StatelessWidget {
       title: 'Plant Expert',
       theme: ThemeData(
         primaryColor: Colors.white,
-        accentColor: Color(0xff119543)
+        accentColor: Color(0xff119543),
       ),
       initialRoute: '/',
-      routes: {
-        '/plant-detail': (context) => PlantDetail(plantInfo: ModalRoute.of(context).settings.arguments),
-        '/login' : (context) => Account(),
-      },
+//      routes: {
```

Comment:
I'm not sure why this was commented out, it worked fine. Now these routes are in the `onGenerateRoute` section, and none of them have a transition. I have also made changes to these routes in the Homescreen branch so we will have to resolve this when merging.